src/sentry/integrations/services/integration/impl.py

```diff
@@ -368,44 +373,43 @@ def send_incident_alert_notification(
         notification_uuid: str | None = None,
     ) -> bool:
         try:
-            sentry_app = SentryApp.objects.get(id=sentry_app_id)
-        except SentryApp.DoesNotExist:
-            logger.info(
-                "metric_alert_webhook.missing_sentryapp",
-                extra={
-                    "sentry_app_id": sentry_app_id,
-                    "organization_id": organization_id,
-                },
+            event = SentryAppEventType(
+                f"metric_alert.{INCIDENT_STATUS[IncidentStatus(new_status)].lower()}"
             )
+        except ValueError as e:
+            sentry_sdk.capture_exception(e)
             return False
 
-        metrics.incr("notifications.sent", instance=sentry_app.slug, skip_internal=False)
+        with SentryAppInteractionEvent(
+            operation_type=SentryAppInteractionType.PREPARE_WEBHOOK,
+            event_type=event,
+        ).capture() as lifecycle:
+            try:
+                sentry_app = SentryApp.objects.get(id=sentry_app_id)
+            except SentryApp.DoesNotExist as e:
+                sentry_sdk.capture_exception(e)
+                lifecycle.record_failure(e)
+                return False
 
-        try:
-            install = SentryAppInstallation.objects.get(
-                organization_id=organization_id,
-                sentry_app=sentry_app,
-                status=SentryAppInstallationStatus.INSTALLED,
-            )
-        except SentryAppInstallation.DoesNotExist:
-            logger.info(
-                "metric_alert_webhook.missing_installation",
-                extra={
-                    "action": action_id,
-                    "incident": incident_id,
-                    "organization_id": organization_id,
-                    "sentry_app_id": sentry_app.id,
-                },
-                exc_info=True,
-            )
-            return False
+            metrics.incr("notifications.sent", instance=sentry_app.slug, skip_internal=False)
 
-        app_platform_event = AppPlatformEvent(
-            resource="metric_alert",
-            action=INCIDENT_STATUS[IncidentStatus(new_status)].lower(),
-            install=install,
-            data=json.loads(incident_attachment_json),
-        )
+            try:
+                install = SentryAppInstallation.objects.get(
+                    organization_id=organization_id,
+                    sentry_app=sentry_app,
+                    status=SentryAppInstallationStatus.INSTALLED,
+                )
+            except SentryAppInstallation.DoesNotExist as e:
+                sentry_sdk.capture_exception(e)
+                lifecycle.record_failure(e)
+                return False
+
+            app_platform_event = AppPlatformEvent(
+                resource="metric_alert",
+                action=INCIDENT_STATUS[IncidentStatus(new_status)].lower(),
```

Comment:
nit: `event` and this `action` param share `INCIDENT_STATUS[IncidentStatus(new_status)].lower()`, so could share a variable there